app/models/route.rb

```diff
@@ -0,0 +1,3 @@
+class Route < ApplicationRecord
+    validates :origin, :destination, presence: true
```

Comment:
Usually origin, destination have a specific format. Like `LHR`, `JFK`. In your case origin and destination do not have any specific format. Is digits allowed in origin and destination?

Reply:
Can you check if empty string or `' '` is accepted as origin and destination? Thank you.

Reply:
added regex for validation. we are using only district name. Do we need formatting origin and destination?